packages/search/src/ago/encode-ago-query.ts

```diff
@@ -0,0 +1,43 @@
+import { IQueryObject } from "./create-q";
+import { calcStart, handleFilter, getSortField } from "./helpers";
+
+export function encodeAgoQuery(queryObject: IQueryObject) {
```

Comment:
i dont think `encode` the right word for this. maybe `concatenate` would be clearer?

Reply:
or may be `createQuery`. I think Daniel had said something like that.